captum/attr/_core/layer/layer_activation.py

```diff
@@ -90,11 +102,12 @@ def attribute(
                 >>> attribution = layer_cond.attribute(input)
         """
         with torch.no_grad():
-            return _forward_layer_eval(
+            layer_eval = _forward_layer_eval(
```

Comment:
is this needed to cover all paths for pyre/mypy?

Reply:
Yup, there was no return outside the with block before.